programs/intent-transfer/src/bridge/config/ntt_config.rs

```diff
@@ -0,0 +1,22 @@
+use crate::error::IntentTransferError;
```

Comment:
i feel like all the config stuff can live in config.rs? tree feels unnecessarily deep otherwise

Reply:
That would break the pattern of having a "processor" file with just the `context` and the `impl` and nothing else for the `register_ntt_config` instruction